metadata.rb

```diff
@@ -14,4 +14,4 @@
 issues_url 'https://github.com/chef-cookbooks/aws/issues'
 chef_version '>= 12.9' if respond_to?(:chef_version)
 
-gem 'aws-sdk', '~> 2'
+gem 'aws-sdk', '>= 2'
```

Comment:
Just to avoid being eventually bitten, maybe this is best written as:

```
gem 'aws-sdk', '>= 2, < 4.0'
```

Reply:
Good point